### ISSUE TITLE
gta3: make any% final split finish after you land

### ISSUE THREAD

**Issue by iguaan** [zoton2/LiveSplit.Scripts#18]
If you MJ at Maria in the end of Exchange, the timer will stop immediately once you trigger her. According to the rules, timer stops once you land and the fade out starts.

**Comment by zoton2** [comment]
Yeah unfortunately back when the values were found, a lot of the newer strats didn't exist (and I'm also not up to date on the GTA3 run anymore too) so you'd have to look at other values you could use.
